deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,177 @@
+# Kubernetes Collection 1.0.0 - Breaking Changes
+
+<!-- TOC -->
+
+- [Helm Users](#helm-users) 
+  - [Changes](#changes) 
+  - [How to upgrade](#how-to-upgrade) 
+    - [1. Upgrade to helm chart version v0.17.1](#1-upgrade-to-helm-chart-version-v0171)
+    - [2. Run upgrade script](#2-run-upgrade-script)
+- [Non-Helm Users](#non-helm-users) 
+  - [Changes](#breaking-changes) 
+  - [How to upgrade](#how-to-upgrade-for-non-helm-users)
+    - [1. Tear down existing collection resources](#1-tear-down-existing-fluentd-prometheus-fluentbit-resources)
+    - [2. Deploy New Resources](#2-deploy-fluentd-fluent-bit-and-prometheus-again-with-the-verison-100-yaml)
+      - [2.1: Deploy Fluentd](#21-deploy-fluentd)
+      - [2.1: Deploy Prometheus](#22-deploy-prometheus)
+      - [2.1: Deploy Fluent-bit](#23-deploy-fluentbit)
+
+
+<!-- /TOC -->
+
+Based on the feedback from our users, we will be introducing several changes to the Sumo Logic Kubernetes Collection solution. Here we detail the changes for both Helm and Non-Helm users, as well as the exact steps for migration.
+## Helm Users
+### Changes
+
+- Falco installation disabled by Default. If you want to enable Falco, modify the `enabled` flag for Falco in `values.yaml` as shown below: 
+```yaml
+falco:
+  ## Set the enabled flag to false to disable falco.
+  enabled: true
+  ```
+
+- Bumped the helm Falco chart version to v `1.1.6` which included a fix to disable the bitcoin/crypto miner rule by default
+
+- Changes in Configuration Parameters
+	- The `values.yaml` file has had several configs moved and renamed to improve usability. Namely, we introduced a new fluentd section into which we moved all of the Fluentd specific configs, while configs for our dependency charts (`prometheus-operator`, `fluent-bit`, `metrics-server`, `falco`) have not changed.
+
+| Old Config 	| New Config 	|
+|:----------------------------------------:	|-----------------------------------------------------	|
```

Comment:
center alignment in one column and left in the second don't look well together in my opinion